src/associations/helpers.ts

```diff
@@ -217,7 +271,11 @@ export function defineAssociation<
   construct: (opts: CleanOptions) => T,
 ): T {
   if (!isModelStatic(target)) {
-    throw new Error(`${source.name}.${lowerFirst(type.name)} called with something that's not a subclass of Sequelize.Model`);
+    throw new Error(
+      `${source.name}.${lowerFirst(
+        type.name,
+      )} called with ${target} that's not a subclass of Sequelize.Model`,
```

Comment:
Main thing was to use `inspect()`, but I did some rephrasing too to add some information:

```suggestion
      )} was called with ${NodeUtils.inspect(target)} as the target model, but it is not a subclass of Sequelize's Model class`,
```

Reply:
It's a very reasonable ask to revert the stylistic changes. I assumed there was a pre-commit hook at work here, but instead it sounds like it was just some aggressive settings in the editor that made the changes. We definitely want to minimize the diff as much as possible :)

Reply:
I have reverted the stylistic changes, should look good now. I also changed the error message to reflect what you had suggested. Thank you for your comments!